contrib/upgrade.sh

```diff
@@ -85,7 +85,7 @@ fi
 # confirm update
 echo "Checking currently installed version..."
 current=$(giteacmd --version | cut -d ' ' -f 3)
-[[ "$current" == "$giteaversion" ]] && echo "$current is already installed, stopping." && exit 1
+[[ "$current" == "$giteaversion" ]] && echo "$current is already installed, stopping." && exit
```

Comment:
```suggestion
[[ "$current" == "$giteaversion" ]] && echo "$current is already installed, stopping." && exit 0
```

I think I prefer explicit status because I don't want to lookup what the default code is for the `exit` builtin.

Reply:
Fair enough. Fixed.